### DISCUSSION TITLE
Update version of streamit when create apps

### DISCUSSION THREAD

**Discussion by Xemnas09** [whitphx/stlite#1003] [General]
Hello. I just want to know if it's possible to update the version of streamlit to the latest (1.36.0). When I'm trying to create any app, I can see the streamlit version is 1.35.0

**Comment by whitphx** [comment]
Unfortunately each Stlite version is bound to a specific version of Streamlit, so you have to wait for a new Stlite release to get the latest Streamlit features.
The latest Stlite is based on Streamlit 1.36.0.
